lib/debug/server_dap.rb

```diff
@@ -656,7 +656,7 @@ def process_protocol_request req
           fail_response req
         end
       else
-        if respond_to? mid = "request_#{req['command']}"
+        if respond_to? mid = "custom_request_#{req['command']}"
```

Comment:
Extension APIs like this should be considered a breaking change and will require bumping a major version number (e.g. `2.0`) as apps and libraries extending `debug` through it will suddenly stop working.
But given now there probably isn't anyone using it yet, it **may** not break anything yet.

Reply:
Now we don't publish this feature in the doc so now it is not public API.